docs/source/features/packaging_output_models.md

```diff
@@ -108,83 +117,92 @@ and for CPU, the best execution provider is CPUExecutionProvider, so the first r
-        * `description [str]`
-          The description for this model registration. This is `None` by default.
-        * `model_package [ModelPackageConfig]`:
-          The configurations for model packaging.
-          * `target_environment [str]`:
-            The environment name for the environment created by Olive. `olive-target-environment` by default.
-          * `target_environment_version [str]`
-            The environment version for the environment created by Olive. Please note if there is already an environment with the same name and the same version in your workspace, your existing environment version will plus 1. This `target_environment_version` will not be applied for your environment. `None` by default.
-          * `inferencing_server [InferenceServerConfig]`
-            * `type [str]`
-              The targeted inferencing server type. `AzureMLOnline` or `AzureMLBatch`.
-            * `code_folder [str]`
-              The folder path to your scoring script.
-            * `scoring_script [str]`
-              The scoring script name.
-          * `base_environment_id [str]`
-            The base environment id that will be used for Azure ML packaging. The format is `azureml:<base-environment-name>:<base-environment-version>`.
-          * `environment_variables [dict]`
-            Env vars that are required for the package to run, but not necessarily known at Environment creation time. `None` by default.
-        * `deployment_config [DeploymentConfig]`
-          The deployment configuration.
-          * `endpoint_name [str]`
-            The endpoint name for the deployment. If the endpoint doesn't exist, Olive will create one endpoint with this name. `olive-default-endpoint` by default.
-          * `deployment_name [str]`
-            The name of the deployment. `olive-default-deployment` by default.
-          * `instance_type [str]`
-            Azure compute sku. ManagedOnlineDeployment only. `None` by default.
-          * `compute [str]`
-            Compute target for batch inference operation. BatchDeployment only. `None` by default.
-          * `instance_count [str]`
-            Number of instances the interfering will run on. `1` by default.
-          * `mini_batch_size [str]`
-            Size of the mini-batch passed to each batch invocation. `10` by default.
-          * `extra_config [dict]`
-            Extra configurations for deployment. `None` by default.
-    * `include_sample_code [bool]`:
-      Whether or not to include sample code in zip file. Defaults to True
-    * `include_runtime_packages [bool]`:
-      Whether or not to include runtime packages (like onnxruntime) in zip file. Defaults to True
+  * `type [PackagingType]`:
+    Olive packaging type. Olive will package different artifacts based on `type`.
+  * `name [str]`:
```

Comment:
What is the default name for DockerFile type?

Reply:
There is no name for Dockerfile type. I can add this explanation to the doc.